lib/auth/userloginstate/generator_test.go

```diff
@@ -219,26 +230,16 @@ func initGeneratorSvc(t *testing.T) (*Generator, *svc) {
 		&svc{AccessLists: accessListsSvc, Access: accessSvc}
 }
 
-func newAccessList(t *testing.T, name string, members []string, roles []string, traits trait.Traits) *accesslist.AccessList {
+func newAccessList(t *testing.T, name string, roles []string, traits trait.Traits) *accesslist.AccessList {
 	t.Helper()
 
-	alMembers := make([]accesslist.Member, len(members))
-	for i, member := range members {
-		alMembers[i] = accesslist.Member{
-			Name:    member,
-			Joined:  time.Now(),
-			Expires: time.Now().Add(24 * time.Hour),
-			Reason:  "added",
-			AddedBy: "owner",
-		}
-	}
-
 	accessList, err := accesslist.NewAccessList(header.Metadata{
 		Name: name,
 	}, accesslist.Spec{
 		Title: "title",
 		Audit: accesslist.Audit{
-			Frequency: time.Hour,
+			Frequency:     time.Hour,
+			NextAuditDate: time.Now().Add(time.Hour * 48),
```

Comment:
This is confusing. If the review frequency is 1 hour why the next audit should be in 48h? :thinking: